Formula/git-imerge.rb

```diff
@@ -1,7 +1,7 @@
 class GitImerge < Formula
   desc "Incremental merge for git"
   homepage "https://github.com/mhagger/git-imerge"
-  url "https://github.com/mhagger/git-imerge/archive/v1.1.0.tar.gz"
+  url "https://github.com/mhagger/git-imerge/archive/v1.2.0.tar.gz"
   sha256 "62692f43591cc7d861689c60b68c55d7b10c7a201c1026096a7efc771df2ca28"
```

Comment:
Thanks for the PR, @mochadwi!

You need to update the `sha256` line as well. It looks like the line should be:

```suggestion
  sha256 "cd80a817bc8dd4b5642f83610237e855fc0a23f5c11978d6f7d021cfb0e87bb2"
```

Additionally, please squash and rename your commit to `git-imerge 1.2.0` (the same as the name of this PR).

Reply:
Well noted with thanks!